easybuild/easyconfigs/g/gtool/gtool-0.7.5-linux_x86_64.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Binary"
+
+name = 'gtool'
+version = '0.7.5'
+versionsuffix = '_linux_x86_64'
+
+homepage = 'http://www.well.ox.ac.uk/~cfreeman/software/gwas/gtool.html'
+description = """ GTOOL is a program for transforming sets of genotype data for use with the programs SNPTEST and IMPUTE.  """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.well.ox.ac.uk/~cfreeman/software/gwas/']
+sources = ['gtool_v%(version)s_x86_64.tgz']
+
+# Following commands are used to untar the file and copy the content to install directory
+install_cmd = "tar xvf gtool_v%(version)s_x86_64.tgz && HOME=%(builddir)s && "
+install_cmd += "cp -a %(builddir)s/* %(installdir)s"
+
+sanity_check_paths = {
+    'files': ["gtool", "LICENCE"],
```

Comment:
this seems to imply that the top-level directory should be added to `$PATH`, to make `gtool` easily available?

This can be done with:

```python
modextrapaths = {'PATH': ''}
```